openlibrary/core/imports.py

```diff
@@ -33,56 +33,59 @@ def load_items(self, filename):
         items = [line.strip() for line in open(filename) if line.strip()]
         self.add_items(items)
 
-    def dedupe_ia_items(self, items):
-        already_present = [
+    def dedupe_items(self, items):
+        ia_ids = [item.get('ia_id') for item in items if item.get('ia_id')]
+        already_present = set([
             row.ia_id for row in db.query(
-                "SELECT ia_id FROM import_item WHERE ia_id IN $items",
+                "SELECT ia_id FROM import_item WHERE ia_id IN $ia_ids",
```

Comment:
I think this is causing a linting issue because python doesn't think `ia_ids` is being used, even though it is as `$ia_ids`

Reply:
Specify `vars` manually instead of using `locals`